charts/platform-storage/templates/admission-controller-deployment.yaml

```
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.admissionController.app_name }}
```

```suggestion
  name: {{ include "platformStorage.fullname" . }}-injector
```
let's use this name for new resources, you can define a template for it